CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ### 0.15.0 (Next)
 
-* [#326](https://github.com/slack-ruby/slack-ruby-client/pull/326): Update new methods - [@wasabigeek](https://github.com/wasabigeek).
+* [#327](https://github.com/slack-ruby/slack-ruby-client/pull/327), [#326](https://github.com/slack-ruby/slack-ruby-client/pull/326): Update new methods - [@wasabigeek](https://github.com/wasabigeek).
```

Comment:
I usually list what has been added, see below, e.g. "Added `admin_conversations_whitelist_add`, ...` or "Added `admin*`", but it's NBD.